build.sbt

```diff
@@ -249,6 +249,13 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
   .settings(
     name := "fs2-core",
     sourceDirectories in (Compile, scalafmt) += baseDirectory.value / "../shared/src/main/scala",
+    unmanagedSourceDirectories in Compile += {
+      val dir = CrossVersion.partialVersion(scalaVersion.value) match {
```

Comment:
Doesn't sbt support this just by naming the source directory `scala-2.13-` (or something like that)? I'm looking at cats: https://github.com/typelevel/cats/tree/master/core/src/main

Reply:
That appears to be done in build.sbt here: 
https://github.com/typelevel/cats/blob/master/build.sbt#L21

Reply:
Never mind then :D